internal/extension/extension.go

```diff
@@ -23,8 +40,10 @@ const (
 	// want to let it having some time for its cold start so we should not set this too low.
 	timeout = 3000 * time.Millisecond
 
-	helloUrl = "http://localhost:8124/lambda/hello"
-	flushUrl = "http://localhost:8124/lambda/flush"
+	helloUrl           = "http://localhost:8124/lambda/hello"
```

Comment:
Since we're no longer calling the hello route, might as well remove all references to it.